common/remote/hooks/git/gitfile.cpp

```diff
@@ -139,6 +139,10 @@ class GitRepositoryFileIO : public CInterface, implements IFileIO
     {
         throwUnexpected();
     }
+    unsigned __int64 getStatistic(StatisticKind kind)
+    {
+        return 0;
```

Comment:
(Note - if the latter, perhaps worth adding a comment. Probably not worth actually implementing though)